projects/plugins/jetpack/changelog/add-new-menu-item-for-inbox-functionality

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Added a new menu item in the left side menu, just after Upgrades item
```

Comment:
@AllTerrainDeveloper -- can you specify that the new menu item is for Inbox? That can't be identified from the changelog otherwise. Additionally, it may be worth noting that the new menu item should only appear for WordPress.com sites.

Reply:
Thank you Dale for your suggestions!